### PR TITLE
Upgrade to LTS 12.9

### PR DESCRIPTION
**Pull request by fsoikin**

### CODE REVIEW

File: graft.cabal [hunk 1]
```diff
@@ -14,31 +14,33 @@ build-type:          Simple
...
+                     , mtl
+                     , text
+                     , containers
+                     , lens
+                     , directory
+                     , parsec
   default-language:    Haskell2010
 
 test-suite graft-test
+  default-extensions:  FlexibleInstances GADTs TemplateHaskell TypeSynonymInstances ExistentialQuantification OverloadedStrings
```
Comment (gasi) [inline graft.cabal]: Nit: Could we list these in alphabetical order so it’s easier to scan (binary search 😉)?

File: graft.cabal [hunk 2]
```diff
@@ -14,31 +14,33 @@ build-type:          Simple
...
-                     , QuickCheck >= 2.8.2
-                     , hspec >= 2.2.4
+                     , mtl
+                     , text
+                     , containers
+                     , lens
+                     , directory
+                     , parsec
+                     , QuickCheck
+                     , hspec
```
Comment (gasi) [inline graft.cabal]: Ditto re: alphabetical order.

**Review by gasi — APPROVED** [review]
LGTM 🚢

File: graft.cabal (hunk 2 above)
Comment (fsoikin) [reply graft.cabal]: Done.

File: graft.cabal (hunk 1 above)
Comment (fsoikin) [reply graft.cabal]: Done.

**Pull request merged**